### ISSUE TITLE
Graphite formatted metric with -1 timestamp

### ISSUE THREAD

**Issue by jenmud** [VictoriaMetrics/VictoriaMetrics#610]
**Describe the bug**
When inserting a graphite metric using the Graphite message format `metric_path value timestamp` and the timestamp is -1, the row is skipped and with the following error message in the logs

```
2020-07-08T08:35:04.507Z        error   VictoriaMetrics/lib/storage/storage.go:1155     warn occurred during rows addition: cannot insert row with too small timestamp -1000 outside the retention; minimum allowed timestamp is 1591518903000
```

This behavior does not conform to the graphite spec outlined [here](https://graphite.readthedocs.io/en/latest/feeding-carbon.html) where is states that if the timestamp is *-1* it will use the time of arrival.

**To Reproduce**
Send a graphite metric with a *-1* timestamp

```
echo "foo.bar.baz;tag1=value1;tag2=value2 123 -1" | nc -N localhost 2003
```

**Expected behavior**
Metric should be inserted into the data with the timestamp when the message arrived and processed by the database.

**Screenshots**

**Version**
```
$ ./victoria-metrics-prod --version
victoria-metrics-20200701-115333-tags-v1.37.4-0-g8da3f773a
```

**Comment by valyala** [comment]
@jenmud , thanks for the bug report. It should be fixed in the commit eb45185eef619d187286a65942b09979fd0921fc . This commit will be included in the next release. In the mean time you may build VictoriaMetrics from this commit according to [these instructions](https://victoriametrics.github.io/#how-to-build-from-sources).

There are the following workarounds for the bug:
* VictoriaMetrics accepts `0` timestamp in Graphite protocol and translates it to the current timestamp.
* VictoriaMetrics accepts Graphite lines without timestamp, i.e. `foo 123` is translated to a `foo` metric with `123` value and with the current timestamp.

**Comment by valyala** [comment]
@jenmud , VictoriaMetrics should properly handle `-1` timestamp in Graphite plaintext protocol starting from [v1.38.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing the issue as fixed.
